extensions/blocks/pinterest/edit.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import { __, _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import { Placeholder, SandBox, Button, IconButton, Toolbar } from '@wordpress/components';
+import { BlockControls } from '@wordpress/block-editor';
```

Comment:
We can't rely on `@wordpress/block-editor` just yet as we still try to support WP 5.1. You should be able to import `BlockControls` from `@wordpress/editor` for now.

```suggestion
import { BlockControls } from '@wordpress/editor';
```